cfgov/unprocessed/js/apps/financial-well-being/fwb-questions.js

```diff
@@ -0,0 +1,83 @@
+/* eslint no-global-assign: "off" */
+'use strict';
+
+var Analytics = require( '../../modules/Analytics' );
+
+function init() {
+
+  var totalQuestions = 12;
+  var quizComplete = false;
+  var questionValues = {};
+  var radioButtons = document.querySelectorAll( '[type="radio"]' );
+  var submitButton = document.querySelector( '#submit-quiz' );
+
+  function allQuestionsCompleted() {
+    return Object.keys( questionValues ).length === totalQuestions;
+  }
+
+  function enableSubmit() {
+    submitButton.title = 'Get your score';
+
+    submitButton.classList.remove( 'a-btn__disabled' );
+  }
+
+  function handleRadio( input ) {
```

Comment:
All methods in this module could use JSDoc comments to describe what they do and their parameters.